src/tools/Reporting/FailureReporting/FailureReporter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Reporting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace FailureReporting
+{
+    public class FailureReporter
+    {
+        public static void CreateFailureReport(string reportJsonPath, Logger logger)
+        {
+            var reporter = Reporter.CreateReporter();
+            if (reporter.InLab && !String.IsNullOrEmpty(reportJsonPath))
+            {
+                File.WriteAllText(reportJsonPath, reporter.GetJson());
+            }
+            logger.Log(reporter.WriteResultTable());
```

Comment:
I think we should remove this, because we will never have any results to write to a table. This also means that you can get rid of the Logger class.

Reply:
agreed